applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes_fractional.h

```diff
@@ -0,0 +1,546 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Uxue Chasco
+//  Co-authors:      Ruben Zorrilla
+//
+
+#pragma once
+#define  KRATOS_TWO_FLUID_NAVIER_STOKES_FRACTIONAL
```

Comment:
```suggestion
```
You no longer need this after the `#pragma once`. Please remove similar ones.

Reply:
Good to know,thank you!